src/components/Modals/TaskStatusUpdate/index.tsx

```diff
@@ -47,7 +47,7 @@ export default function TaskStatusUpdate() {
 
   return (
     <TaskStatusUpdatePresentation
-      isOpen={isTaskNoteworthy}
+      isOpen={true}
```

Comment:
we have the condition in layout as well for rendering the component, so it doesn't make sense to have this at taskNoteworthy visiblity here

NOTE: the state used here was incorrect